src/nvim/globals.h

```diff
@@ -72,7 +72,7 @@ EXTERN long Rows                        /* nr of rows in the screen */
   = 24L
 #endif
 ;
-EXTERN long Columns INIT(= 80);         /* nr of columns in the screen */
+EXTERN int Columns INIT(= 80);         /* nr of columns in the screen */
```

Comment:
Forget the above. Everything I said is true if you could change `Columns`, but the fact is that you can't, because it's an option too, so it must be `long`, like the other ones below (if you haven't read comment on options below, please do). The fact that `Columns` doesn't follow the `p_xxxx` naming convention, as the other ones, confused me.